backend/main.py

```diff
@@ -52,8 +54,8 @@
     return dict
 
 
-@app.post("/write_events")
-async def write_event(event):
+@app.route("/write_events", methods=['POST'])
```

Comment:
On line 31 you added a slash after "/read_events**/**", so it is good to stay consistent and add it here as well. Also, keep in mind that the slash does make a difference for functionality as well. This is because you (the front end folks) need to explicitly make the same exact call written in the code. I learned this just a few days ago lol, but it was awful trying to figure out what was wrong.

Reply:
Also, I think this may have been my fault as well when I did this... but I think it's good info to know and thanks for fixing it!

Reply:
Would it make more sense to add the slash after the write_events or delete the slash after read_events?

Reply:
Let's just delete the slash after read_events... I guess it's a personal preference